tests/functional/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -114,9 +123,12 @@ def test_non_ocs_taint_and_tolerations(self):
             )
         else:
             param = (
-                f'{{"spec": {{"placement": {{"all": {tolerations}, "mds": {tolerations}, '
-                f'"noobaa-core": {tolerations}, "rgw": {tolerations}}}}}}}'
+                f'"all": {tolerations}, "mds": {tolerations}, '
+                f'"noobaa-core": {tolerations}, "rgw": {tolerations}, '
             )
+            if version.get_semantic_ocs_version_from_config() >= version.VERSION_4_15:
```

Comment:
We don't need this condition as we are working with release branches. Just backport the fix till ODF 4.15. The current master will become release branch for 4.16.